src/core/DatePicker/DatePicker.tsx

```diff
@@ -124,7 +125,13 @@ export type DatePickerProps = {
    * @default false
    */
   setFocus?: boolean;
-} & Omit<CommonProps, 'title'>;
+  /**
+   * Show time picker near the calendar.
+   * @default false
+   */
+  showTime?: boolean;
+} & Omit<CommonProps, 'title'> &
```

Comment:
Think `Omit<CommonProps, 'title'` is not needed anymore as `StylingProps` are coming from `TimePicker`

Reply:
oh, that got a little messy...

Reply:
removed..